modules/test/playwright/helpers/ApiHelpers.ts

```diff
@@ -322,6 +326,23 @@ export class ApiHelpers {
 		return response.json();
 	}
 
+	async patchMultipart<T>(url: string, options: PostOptions<T> = {}) {
+		const response = await this.page.request.patch(url, {
+			data: options.data,
+			failOnStatusCode: options.failOnStatusCode || false,
+			headers: options.headers || (await getHeader(this.page)),
+			multipart: options.multipart,
+		});
+
+		const text = await response.text();
+
+		if (!text) {
+			return response;
+		}
+
+		return response.json();
+	}
+
```

Comment:
> Can you create an new interface `PatchOptions?` Alternatively, maybe it can be renamed simply to `Options` or something like that.

Yes, a unique `RequestOptions` would fit for `post`, `put` and `patch` methods. I've checked in the playwright `APIRequestContext` interface that `options` is separately specified for each HTTP method, but all declarations are identical .

Reply:
Sounds good. Bear in mind soon this will change anyway: https://liferay.atlassian.net/browse/LPD-41174.

Reply:
@adolfopa I've added a new commit to change the options name [738303e](https://github.com/liferay-content-management/liferay-portal/pull/6135/commits/738303e580b065f3d8cff795e177006921b8319c)

If you agree I will not remove the old `patch` method because I would need to test all 17 changes because this old method has also some logic inside :sweat:

Reply:
If you want to keep the method, you need to find another name for it. It doesn't do a multipart patch, it does whatever its options specify.

Anyway, I think it is easier to change the 17 occurrences and run the playwright suites for Commerce, User and Taxonomy. Those are the only 3 apps that would be affected by the change, so if those pass you're done. You don't need to do any additional testing.